operator/cmd/main.go

```diff
@@ -145,9 +147,17 @@ func main() {
 		metricsServerOptions.FilterProvider = filters.WithAuthenticationAndAuthorization
 	}
 
+	controllerOpts := config.Controller{
+		MaxConcurrentReconciles: 25,
+	}
+	if s, ok := os.LookupEnv("MAX_CONCURRENT_RECONCILES"); ok {
+		controllerOpts.MaxConcurrentReconciles, _ = strconv.Atoi(s)
```

Comment:
Should probably panic if this doesn't parse, otherwise it'll be zero which I'm guessing would do... nothing?